R/derive_extreme_event.R

```diff
@@ -347,18 +347,15 @@ derive_extreme_event <- function(dataset,
                                  order,
                                  mode,
                                  source_datasets = NULL,
-                                 ignore_event_order = FALSE,
+                                 ignore_event_order = NULL,
                                  check_type = "warning",
                                  set_values_to,
                                  keep_source_vars = exprs(everything())) {
   # Check input parameters
+  assert_data_frame(dataset, optional = TRUE)
```

Comment:
The default value of `dataset` (in line 343) should be set to `NULL`.

Reply:
Updated, thanks!